addons/pos_restaurant/static/tests/tours/helpers/ProductScreenTourMethods.js

```diff
@@ -104,3 +104,17 @@ export function orderBtnIsPresent() {
         },
     ];
 }
+export function tableNameShown(table_name) {
+    return [
+        {
+            content: "Table name is shown",
+            trigger: ".table-name",
```

Comment:
You can include the table number in the trigger. Something like `.table-name:contains${table_name}` should work